README.md

```diff
@@ -56,7 +56,6 @@ In the Kafka world nothing is easy, but esque (pronounced esk) is an
 attempt at it.
 
 Options:
---recreate-config   Overwrites the config with the sample config.
```

Comment:
please also add change to `edit` here.

Reply:
I am going to change it in the help text branch because there are also changes there to the `esque` output